frontend/src/api/LoanApi.js

```diff
@@ -0,0 +1,12 @@
+export default {
+    getLoans() {
+        return fetch('http://localhost:8000/api/loan/')
```

Comment:
I don't like hard-coding the api url, but didn't seem like there was an easy method that I could use for a quick win here. Anyone know of a more dynamic way to get this?